pkg/scheduler/apis/config/v1beta3/default_plugins.go

```diff
@@ -166,9 +100,15 @@ func mergePluginSet(defaultPluginSet, customPluginSet v1beta3.PluginSet) v1beta3
 	enabledCustomPlugins := make(map[string]pluginIndex)
 	// replacedPluginIndex is a set of index of plugins, which have replaced the default plugins.
 	replacedPluginIndex := sets.NewInt()
+	var disabled []v1beta3.Plugin
```

Comment:
BTW: how about renaming it to `disabledPlugins`, and rename current `disabledPlugins` to `disabledPluginNames`?

Reply:
Good point, in the past we didn't care about disabled plugins in the default set because ot held no meaning really since one can just remove it from the enabled set. With multipoint we could give it a meaning.

Reply:
There are no disabled plugins in the default plugin set, the only one we had was VolumeBinding. So, I removed the logic to handle that in https://github.com/kubernetes/kubernetes/pull/105611/commits/8a81833da8c9a48b65ced41a00ad8289cccf7f63#diff-f1c8947cb8d7c33e8b99363e4002b12fbdf0bed7fb11eb857529204469b9e769L111-L112

I can re-add the logic in case we do add disabled plugins in the future, but for now it would do nothing.

There is a unit test for disabling a default plugin manually: https://github.com/kubernetes/kubernetes/blob/a11794e34823bfd6fd1159261cda18a1335cbd9f/pkg/scheduler/apis/config/v1beta3/default_plugins_test.go#L390

Reply:
ok, since this logic is versioned, we can keep it as is because there are no disabled extensions in the default set of plugins. Changing the default set requires upping the version, and only then we need to handle this if we ever wanted to. 

I am ok with both adding it back or keeping it as is, we are safe in both cases.

Reply:
I re-added the logic to handle this with a test case just in case we do add default disabled plugins in the future.

Reply:
> I re-added the logic to handle this with a test case just in case we do add default disabled plugins in the future.

Thanks. The latest code looks good.